package.json

```diff
@@ -23,13 +23,16 @@
     "url": "https://github.com/geckoboard/geckoboard-node/issues"
   },
   "homepage": "https://developer-beta.geckoboard.com/js",
```

Comment:
This is wrong, it should be developer.geckoboard.com

Reply:
Line 21 (`license`) is also wrong — which reminds me that we need to choose a licence for this library.